sdk/nanotdf_config.go

```diff
@@ -30,7 +30,7 @@ type NanoTDFConfig struct {
 type NanoTDFOption func(*NanoTDFConfig) error
 
 // NewNanoTDFConfig - Create a new instance of a nanoTDF config
-func (s SDK) NewNanoTDFConfig() (*NanoTDFConfig, error) {
```

Comment:
This is already used in DSP so you'll have to mark it as deprecated and not rename it